src/main/java/hudson/plugins/testlink/AbstractTestLinkBuilder.java

```diff
@@ -182,13 +188,13 @@ public AbstractTestLinkBuilder(String testLinkName, String testProjectName, Stri
      * @deprecated
      */
     public AbstractTestLinkBuilder(String testLinkName, String testProjectName, String testPlanName,
-            String platformName, String buildName, String customFields, Boolean executionStatusNotRun,
+            String platformName, String buildName, String customFields, String testPlanCustomFields, Boolean executionStatusNotRun,
```

Comment:
I come from the C# world, all those deprecated tons of code and almost empty methods and constructors look really frustrating to me (since java does not have default param value in method declaration) 
I thought that we can just replace it and if something brakes we have git history and smart IDEs to handle that,